packages/storybook/src/documentation/pages/pagination.stories.mdx

```diff
@@ -90,10 +107,45 @@ To achieve this, just set the pagination property of the `components` react prop
 
 <CustomPaginationDemo />
 
-## Pagination API
+## Server-side Pagination
+
+XGrid supports both client and server side pagination. By default, pagination works on the client side. To switch it to server side,
+set the property `paginationMode` to `Server` as string or you can use our `FeatureMode` enum.
+**Note**: You also need to set the `rowCount` property to override the number of rows in the grid so XGrid can calculate the total number of pages.
+
+Finally, you're going to need to handle the `onPageChanged` event, to load the rows for the corresponding page.
+
+```tsx
+const [rows, setRows] = useState<RowsProp>([]);
```

Comment:
```suggestion
const [rows, setRows] = React.useState<RowsProp>([]);
```

Reply:
It needs to be done properly, you need to clean up the imports...
I would do it in a separate PR on the whole repo, big bang change...